comfort-service.yml

```diff
@@ -0,0 +1,291 @@
+#
```

Comment:
This is just a copy of the file in VSC repo. Should we consider having the same "submodule" setup as VSC

Reply:
@erikbosch As an alternative the vsc checkout could also only happen during the test in the CI, since git submodules can be unhandy.